### DISCUSSION TITLE
Passaggio anno successivo

### DISCUSSION THREAD

**Discussion by scuolagioco** [iisgiua/giuaschool#269] [Richieste di aiuto] [answered]
Salve, c'è qualche procedura da effettuare per poter fare il passaggio all'anno successivo?

**Comment by trinko** [comment, marked as answer]
Attualmente si prevede una installazione differente per ogni anno scolastico:

- a fine anno si fa la procedura di archiviazione (**SISTEMA > ARCHIVIAZIONE**) che prevede la generazione dei documenti in PDF dei vari registri ed eventuali altri documenti; i documenti sono generati sul server e dovranno poi essere scaricati dalle cartelle **FILES/archivio**
- si fa poi una nuova installazione per il nuovo A.S. e si esegue la procedura di importazione  (**SISTEMA/IMPORTAZIONE INIZIALE**) che carica i dati dal database dell'anno precedente.

Le procedure sono da rivedere: conto di metterci le mani appena posso, per rendere le procedure più generiche, creando una vera esportazione di tutti i dati e una importazione su qualsiasi altro sistema.
